java/ql/lib/semmle/code/java/security/ExternallyControlledFormatStringQuery.qll

```diff
@@ -0,0 +1,26 @@
+/** Provides a taint-tracking configuration to reason about externally controlled format string vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.StringFormat
```

Comment:
I think it's a good moment to make all these imports private (since `*Query.qll` files will be importable). If doing so makes the `.ql` files need extra imports, add them there as well.